package.json

```diff
@@ -17,7 +17,7 @@
     "@graasp/sdk": "4.29.1",
     "@graasp/stylis-plugin-rtl": "2.2.0",
     "@graasp/translations": "1.37.1",
-    "@graasp/ui": "5.0.1",
+    "@graasp/ui": "5.2.1",
```

Comment:
In this update from UI, the ThemeProvider accepts `ReactNode` as children which is ok with supplying `null`. Before this was not possible.